.gitlab-ci.yml

```diff
@@ -0,0 +1,16 @@
+# See https://docs.gitlab.com/ce/ci/yaml/README.html for the documentation
+
+package:
+  image: debian:10
+  stage: build
+  before_script:
+    - apt update
+    - apt install --yes python3-pip
```

Comment:
When used in non-interactive way, apt usually prints a warning:
```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
```
If you decide to keep installing python by yourself, please consider changing to apt-get.

Reply:
Done.